iroha-transfer-asset/intro.md

```diff
@@ -1,8 +1,10 @@
-This is a Linux-based sandbox that runs an instance of Hyperledger Iroha v1.0 and demonstrates how users can interact with it's API using Python library.
+This is a Linux-based sandbox that runs Hyperledger Iroha v1.0. 
```

Comment:
I suggest to mention that this installation of iroha consists of a single node.

Reply:
fixed